### ISSUE TITLE
Mark deprecated & archive repo

### ISSUE THREAD

**Issue by gwern** [baumgarr/Nixnote2#517]
Development of this package seems permanently abandoned for the past 5 years or so, and it no longer works with Evernote with the latest change of ownership & API changes, AFAICT. The successor everyone seems to be using is the [robert7](https://github.com/robert7/nixnote2) fork, which has been maintained up until recently and still works with Evernote (as I've verified just now by upgrading). However, people/search engines are still pointing to this repo and the [Sourceforge repo](https://sourceforge.net/projects/nevernote/) (which claims that 20 people are downloading it every week still!).

Considering that, so users don't waste time installing or running or filing more bug reports on an obsolete & abandoned repo, all this should be tidied up & formalized: the README of this repo & the Sourceforge should be updated to point to the robert7 repo, and then repos archived/made read-only.

We salute the codebase & developer for their service all these years to Evernote users like us (ensuring, if nothing else, that there was always a working escape hatch for our notes), and wish them a happy retirement.

**Comment by robert7** [comment]
I did not hear from the original maintainer (Randy Baumgarte) since 2018, so unlikely he will answer.
Unfortunately, I neither have now time to continue development of my fork, but it should work still quite fine.

**Comment by gwern** [comment]
(It's my hope that Randy can take a few minutes to properly close down the repo and save himself any annoyance from additional notifications/people bugging him; if he can't, then this issue is still useful because people will click on 'Issues' and see "Mark deprecated & archive repo" at the top of the list and immediately understand the situation.)

**Comment by robert7** [comment]
Yes, that would be nice, but I'm afraid that Randy doesn't respond since years because he is not reachable for what ever reason.
